### PR TITLE
Walls

### PR DESCRIPTION
**Pull request by peregrinebalas**
Adds
Pipfile from pipenv install
django-rest-framework package
serializer for Wall
View for POST to api/v1/walls
updates to settings for rest framework config

### CODE REVIEW

**Comment by peregrinebalas** [comment]
image

File: scrawl_be/settings.py [hunk 1]
```diff
@@ -107,7 +120,10 @@
     },
 ]
 
-
+CORS_ORIGIN_ALLOW_ALL = False
+CORS_ORIGIN_WHITELIST = [
+    'http://localhost:8000'
+]
```
Comment (smthom05) [inline scrawl_be/settings.py:126]: I assume we can just add on our Heroku FE when it's up and running?

File: scrawl_be/settings.py [hunk 2]
```diff
@@ -88,7 +91,17 @@
...
-
+REST_FRAMEWORK = {
+    'DEFAULT_RENDERER_CLASSES': (
+        'rest_framework.renderers.JSONRenderer',
+    ),
+    'DEFAULT_PARSER_CLASSES': (
+        'rest_framework.parsers.JSONParser',
+    ),
+    'DEFAULT_PERMISSION_CLASSES': (
+        'rest_framework.permissions.AllowAny',
```
Comment (smthom05) [inline scrawl_be/settings.py:102]: What is this allowing? I see it referenced in our `scrawls/views.py` file down below.

File: scrawls/views.py [hunk 3]
```diff
@@ -1,3 +1,32 @@
...
+
+class CreateWall(generics.CreateAPIView):
+    permission_classes = (permissions.AllowAny,)
+
+    def post(self, request, *args, **kwargs):
+        name = request.data.get("name", "")
+        address = request.data.get("address", "")
+        lat = request.data.get("lat", "")
+        lng = request.data.get("lng", "")
+        range = request.data.get("range", "")
```
Comment (smthom05) [inline scrawls/views.py:17]: Is `range` a reserved word in python? Curious why it's showing up in blue here...

**Review by smthom05 — APPROVED** [review]
This looks great. Added a few questions I had about just general implementation, if you have time to look through them.

File: scrawl_be/settings.py (hunk 1 above)
Comment (peregrinebalas) [reply scrawl_be/settings.py:126]: You got it!

File: scrawl_be/settings.py (hunk 2 above)
Comment (peregrinebalas) [reply scrawl_be/settings.py:102]: This is a CORS thing, I set it at the moment to just allow any request. We should probably change it later.

File: scrawls/views.py (hunk 3 above)
Comment (peregrinebalas) [reply scrawls/views.py:17]: Good question, I don't think so, but maybe. It's working at the moment but there's probably something to look into there.

**Pull request merged**